src/firecfg/firecfg.config

```diff
@@ -683,6 +683,7 @@ plv
 pngquant
 polari
 porn-cli
+Postman
```

Comment:
The firecfg.config file is sorted case-sensitively. Move up `Postman` to line 26...